specificatie/genereervariant/openapi.yaml

```diff
@@ -33,7 +33,7 @@ paths:
         1.  Raadpleeg met burgerservicenummer
         2.  Zoek met geslachtsnaam en geboortedatum
         3.  Zoek met geslachtsnaam, voornamen en gemeente van inschrijving
-        4.  Zoek met postcode en huisnummer
+        4.  Zoek met postcode, huisnummer en geboortedatum
```

Comment:
bij de zoekingangen noemen we verder alleen de verplichte (minimale) parameters, nu heb je ook geboortedatum in de beschrijving toegevoegd. Die is optioneel, net als o.a. huisletter. Dus ik denk dat je het postcode en huisnummer kan laten